docs/UserGuide.md

```diff
@@ -197,7 +196,7 @@ Examples:
 *  `edit 1 d/Must include username, email and password fields by/15-12-2020 12:00`
 edits the description of the 1st deliverable to be `Must include username, email and password fields`
 and its deadline to be `15-12-2020 12:00`.
-*  `edit 2 c/Betsy Crowe, Jeremey` edits the contacts of the 2nd deliverable to be `Betsy Crowe, Jeremey`.
```

Comment:
no harm keeping

Reply:
not very in favour of cluttering the UG/MSS steps. can discuss again with the rest during meeting